server/tests/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+include_directories (../)
+
+set(TEST_SRCS host/ enclave/)
+
+add_executable(Test host_test.cpp)
+
+find_library(MBEDTLS mbedtls)
+target_link_libraries(Test PRIVATE modelaggregator_host mbedcrypto mbedtls)
```

Comment:
Do you need to link against mbedtls and mbedcrypto in this cmake?

Reply:
yup, get undefined references otherwise